sources/editor/Stride.Core.Assets.Editor/View/AssetViewUserControl.xaml.cs

```diff
@@ -5,7 +5,7 @@
 using System.Windows;
 using System.Windows.Controls;
 using System.Windows.Input;
-using Xceed.Wpf.Toolkit.Core;
+//using Xceed.Wpf.Toolkit.Core;
```

Comment:
Better remove code instead of commenting out.

Reply:
Ok, will update this and the other one.